packages/whemotion-admin/src/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import logo from './logo.svg'
+import { AppHeader, AppLink, AppLogo, AppWrapper } from './App.style'
+import { Button } from '@yisuses/whemotion-common'
+
+export const App: React.FC = () => (
```

Comment:
I guess we will only use FC when children are needed. Otherwise we should use `React.ReactElement`